src/nn/ops/conv2d.ts

```diff
@@ -256,6 +256,9 @@ export class Conv2d extends SingleOutputOperation implements FusedOperation {
           input.shape[2] * this.strides_[1],
         ];
       }
+      // tf.conv2dTranspose filter layout (hwoi): [filterHeight, filterWidth,
+      // outDepth, inDepth]
+      filter = tf.transpose(filter, [0, 1, 3, 2]);
```

Comment:
Thanks @BruceDai , however, could you please transpose the filter for tf.conv2dTranspose around L.176 together with tf.conv2d. With that, the implementation could avoid transposing filters twice for conv2dTranspose. Does it make sense?

Reply:
@huningxin Yes, please take another look at new commit thanks.